libs/designsystem/src/lib/components/button/button.component.spec.ts

```diff
@@ -273,9 +273,9 @@ describe('ButtonComponent', () => {
       });
     });
 
-    describe('when configured with attentionlevel 4', () => {
+    describe('when configured with no decoration', () => {
```

Comment:
Consider adding back the 'configured with attentionlevel 4' test, and test that the `isAttentionLevel3` variable is correctly set when level 4 is used as input.

Reply:
-reinserted